utils/sound_upload.py

```diff
@@ -173,7 +174,7 @@ def create_sound(user, sound_fields, apiv2_client=None, process=True, remove_exi
             sound.geotag = geotag
 
     # 6 set description, tags
-    sound.description = sound_fields['description']
+    sound.description = remove_control_chars(sound_fields['description'])
```

Comment:
this was removed when we merged these 2 methods, so I'm restoring it (we discussed that perhaps it wasn't needed because we also call it when adding sound to solr, but I think that it's also good to have this fixed in the database)

Reply:
I'm still not sure about this one. Why would be want to have this fixed in DB for sound descriptions and not for, e.g., comments or any other text field? If we only use `remove_control_chars` for this field, I think we should add a comment about why here and not in other places.

Reply:
I raised this as #1322